### PR TITLE
[JBEAP-11776] ARTEMIS-1257 - IJ020017: Invalid archive: file://RaActi…

### PR DESCRIPTION
**Pull request by xstefank**
…vatoractivemq-ra when pooled connection factory rebalance-connections is set to true

ARTEMIS: https://issues.apache.org/jira/browse/ARTEMIS-1257
JBEAP: https://issues.jboss.org/browse/JBEAP-11776

Upstream PRs: https://github.com/apache/activemq-artemis/pull/1371, https://github.com/apache/activemq-artemis/pull/1374

### CODE REVIEW

File: artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java [hunk 1]
```diff
@@ -623,7 +623,7 @@ public void setUseLocalTx(final Boolean localTx) {
       this.localTx = localTx;
    }
 
-   public boolean isRebalanceConnections() {
+   public Boolean isRebalanceConnections() {
```
Comment (jmesnil) [inline artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java]: You should make the same change for the setter `setRebalanceConnections`

**Review by jmesnil — CHANGES_REQUESTED** [review]

File: artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java (hunk 1 above)
Comment (xstefank) [reply artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java]: Thank you, should be done now.

**Review by jmesnil — APPROVED** [review]

**Comment by jmesnil** [comment]
@xstefank you should also open this PR against Artemis upstream 1.x branch

**Comment by jmesnil** [comment]
PR for Artemis upstream master branch https://github.com/apache/activemq-artemis/pull/1371

**Comment by andytaylor** [comment]
@xstefank you need these fixes at apache first, once they are merged you need to cherry-pick them with -x flag and add the EAP Jira to the commit message

**Comment by xstefank** [comment]
@andytaylor should I close this one?

**Comment by andytaylor** [comment]
close it or just change it and do a forced push, its up to you

**Comment by clebertsuconic** [comment]
@jmesnil how could this happen? just the Boolean by boolean? wouldn't the RA be able to handle such things?

**Comment by jmesnil** [comment]
@clebertsuconic you will have to ask to @xstefank as he provided the patch. I just commented that for consistency sake, the setter should use the same type than the getter.

**Pull request closed without merging**